specification/resources/resource-manager/Microsoft.Resources/stable/2020-10-01/subscriptions.json

```diff
@@ -0,0 +1,730 @@
+        },
+        "countryCode": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Country/region abbreviation for the tenant."
+        },
+        "displayName": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The display name of the tenant."
+        },
+        "domains": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "type": "string",
+            "description": "The domain name for a tenant."
+          },
+          "description": "The list of domains for the tenant."
+        },
+        "defaultDomain": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The default domain for the tenant."
+        },
+        "tenantType": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The tenant type. Only available for 'Home' tenant category."
+        },
+        "tenantBrandingLogoUrl": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The tenant's branding logo URL. Only available for 'Home' tenant category."
+        }
+      },
+      "description": "Tenant Id information."
+    },
+    "TenantListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/TenantIdDescription"
+          },
+          "description": "An array of tenants."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URL to use for getting the next set of results."
+        }
+      },
+      "required": [
+        "nextLink"
+      ],
+      "description": "Tenant Ids information."
+    },
+    "ResourceName": {
+      "description": "Name and Type of the Resource",
+      "required": [
+        "name",
+        "type"
+      ],
+      "properties": {
+        "name": {
+          "description": "Name of the resource",
+          "type": "string"
+        },
+        "type": {
+          "type": "string",
+          "description": "The type of the resource"
+        }
+      }
+    },
+    "CheckResourceNameResult": {
+      "properties": {
+        "name": {
+          "description": "Name of Resource",
+          "type": "string"
+        },
+        "type": {
+          "description": "Type of Resource",
+          "type": "string"
+        },
+        "status": {
+          "description": "Is the resource name Allowed or Reserved",
+          "type": "string",
+          "enum": [
+            "Allowed",
+            "Reserved"
+          ],
+          "x-ms-enum": {
+            "name": "ResourceNameStatus",
+            "modelAsString": true
+          }
+        }
+      },
+      "description": "Resource Name valid if not a reserved word, does not contain a reserved word and does not start with a reserved word"
+    },
+    "ErrorResponse": {
```

Comment:
You should use the common-types error definition, which matches this one.

Reply:
updated